### ISSUE TITLE
[Bug]: Building Placer - Construction not saved to server anywhere except HQ

### ISSUE THREAD

**Issue by TheDarcLord** [official-antistasi-community/A3-Antistasi#3065]
### Describe the bug

The Building Placer works well! 
However, structures are only saved at the HQ upon server shutdown. Any other emplacements are deleted. This includes emplacements at captured outposts, bases, seaports, etc. 

### How to reproduce

1. Build a bunker at the HQ
2. Build a bunker at an Outpost
3. Shutdown the Server
4. Relaunch
5. Fast Travel to both locations - Bunker at Outpost will be missing

### Version

3.4.0

### Have you altered the code?

No

### Map

Altis

### What server?

LAN Hosted

### Mods

```html
None
```

**Comment by TheDarcLord** [comment]
https://ko-fi.com/home/coffeeshop?txid=840b8e80-df95-44a2-93f2-5b741623b466&mode=public&img=ogiboughtsomeone

**Comment by Bob-Murphy** [comment]
That's bribery.



I like that!

**Comment by jaj22** [comment]
Nah, it's working correctly. Because there's currently no spawning or despawning of these buildings, I temporarily limited saving to mission radius from HQ for performance reasons. So constructions at outposts within 4km (by default) will save, and others won't. Sucks if you're defending an airbase on the other side of the map, but compromises were required.

**Comment by jaj22** [comment]
New-garrison PR now saves buildings everywhere if they're on a rebel marker.

**Comment by jaj22** [comment]
Fixed in 3.10 somewhere.

**Comment by TheDarcLord** [comment]
You took a while to finish that Coffee...
